src/dataServices/dynamoDbDataService.ts

```diff
@@ -139,19 +145,88 @@ export class DynamoDbDataService implements Persistence {
         };
     }
 
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
     async initiateExport(initiateExportRequest: InitiateExportRequest): Promise<string> {
-        throw new Error('method not implemented');
+        await this.throttleExportRequestsIfNeeded(initiateExportRequest.requesterUserId);
+        // Create new export job
+        const jobId = uuidv4();
+
+        // TODO: Start Export Job Step Function
+        // const stepFunctionArn = await StartStepFunctionAndGetStepFunctionArn
+
+        const params = DynamoDbParamBuilder.buildPutCreateExportRequest(jobId, initiateExportRequest, '');
+        await this.dynamoDb.putItem(params).promise();
+        return jobId;
+    }
+
+    async throttleExportRequestsIfNeeded(requesterUserId: string) {
+        const MAXIMUM_SYSTEM_LEVEL_CONCURRENT_REQUESTS = 2;
+        const MAXIMUM_CONCURRENT_REQUEST_PER_USER = 1;
+
+        const queryJobStatusParam = DynamoDbParamBuilder.buildQueryExportRequestJobStatus('in-progress');

```

Comment:
should we account for the 'cancelling' status?

Reply:
Yeah, let's update this to include the `canceling` status. Otherwise a user could have a large number of jobs in the canceling state at a time. 
When a job is in the `canceling` status, the Export job step function should pick up the new state change and start canceling the Glue job. The rate at which this happens depends on how often we want the step function to check for job status. Maybe once every 3 minutes? As soon as the job is in the canceling state, the Glue job should no longer consume DDB read units.  However, for those 3 minutes, the Glue jobs will still be consuming DDB read units.

Reply:
I think that we should check the job cancelling status a lot more often than 3 min. Maybe 10 seconds. 

Checking the job status is cheap(just a ddb query). Export jobs will likely consume thousands of RCU, it's more efficient to poll for status frequently to cancel them ASAP.

Reply:
Yep, we can do a shorter time. The timing for how often the job status isn't defined in this code base. But for our step function we can set the job status check time to be shorter.